packages/shared/src/hooks/log/useLogQueue.ts

```diff
@@ -72,6 +81,10 @@ export default function useLogQueue({
       },
       queueRef,
       sendBeacon: () => {
+        if (isFirefox && permission !== FirefoxPermissionType.Accepted) {
+          return;
+        }
```

Comment:
@idoshamun we are now stopping the send of analytics until it is accepted, only of course for Firefox.

Reply:
Yes. The `isFirefox` variable checks the `process.env.TARGET_BROWSER` which is only applied to extension builds.

Reply:
Perfect!

Reply:
Let's change the naming in that case, you can already see it's confusing :)